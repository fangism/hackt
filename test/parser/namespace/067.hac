// "067.hac"
// test case for improving error message
// contributed by Carlos Tadeo Ortega Otero

namespace inverter { }

defproc inverter ( bool a, b ) { }

inverter i;

