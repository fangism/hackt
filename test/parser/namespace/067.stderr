Identifier `inverter' already taken by entity::namespace
ERROR: Failed to add definition! [7:1..34]
